yb-voyager/cmd/analyzeSchema.go

```diff
@@ -469,15 +468,10 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 		} else if tbl := likeRegex.FindStringSubmatch(sqlInfo.stmt); tbl != nil {
 			summaryMap["TABLE"].invalidCount[sqlInfo.objName] = true
 			reportCase(fpath, "LIKE clause not supported yet.",
-				"https://github.com/YugaByte/yugabyte-db/issues/1129", "", "TABLE", tbl[2], sqlInfo.formattedStmt)
-		} else if tbl := tblPartitionRegex.FindStringSubmatch(sqlInfo.stmt); tbl != nil {
-			tblParts[tbl[2]] = tbl[3]
-			if filename, ok := primaryCons[tbl[2]]; ok {
-				reportAddingPrimaryKey(filename, tbl[2], sqlInfo.formattedStmt)
-			}
```

Comment:
why removed this check? and regex tblPartitionRegex

Reply:
Handled it with other Partitions regex and case below, `partitionColumnsRegex`